db/re/item_combo_db.conf

```diff
@@ -1073,7 +1073,7 @@ combo_db: (
 {
 	Items: ["Elite_Shooter_Suit", "Battle_Boots", "Sheriff_Manteau"]
 	Script: <"
-		bonus2 bSubRace, RC_NonDemiPlayer, -300;
+		bonus2(bAddRaceTolerance, RC_NonDemiPlayer, -200);
```

Comment:
is there a reason to change -300 into -200 ?

Reply:
Yes, `` is wrong and `-200` is right. :grin:
```
item COMBIITEM_Elite_Shooter_Set
	event OnStartEquip:
		SubRaceTolerace RACE_All 200
		AddRaceTolerace RACE_Human 200
		AddRaceTolerace RACE_Player 200
		AddExtParam User VAR_MAXHPPERCENT 12
		AddExtParam User VAR_DEXAMOUNT 3
		SubRangeAttackDamage User 10
		SubSpellDelay 25
	return
	event OnFinishEquip:
		AddRaceTolerace RACE_All 200
		SubRaceTolerace RACE_Human 200
		SubRaceTolerace RACE_Player 200
		SubExtParam User VAR_MAXHPPERCENT 12
		SubExtParam User VAR_DEXAMOUNT 3
		AddRangeAttackDamage User 10
		AddSpellDelay 25
	return
```